dist/placeholder-buster.txt

```diff
@@ -51,6 +51,20 @@ dailymotion.com##div[class^="AdDiscovery"]
 # https://github.com/NanoAdblockerLab/NanoContrib/pull/16#issuecomment-473483169
 myspace.com##.container-main > ul > li:has(div[id="ad-placement"])
 
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/17
+!#if env_mobile
+instagram.com##footer:style(margin-top: 25px !important)
+!#endif
+
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/17
+techadvisor.*##.adsbygoogle
+techadvisor.*###topLeaderBoardHolder
+techadvisor.*###marketPlace
+techadvisor.*###bottomLeaderBoardHolder
+
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/17
+change.org##p:has-text(/^\xA0$/):not(:has-text(/[a-z]/i))
```

Comment:
This is out of scope, as it's not ads placeholders. Authors padding their posts has nothing to do with ads.

Reply:
I could've tried to make a list of my own that deals with such things, but it'd be a low priority of mine for at least the next few weeks.